src/main/webapp/js/feedbackSubmissionsEdit.es6

```diff
@@ -1,93 +1,100 @@
+const WARNING_STATUS_MESSAGE = '.alert-warning.statusMessage';
 
 // text displayed to user
-var SESSION_NOT_OPEN = 'Feedback Session Not Open';
-
-$(document).ready(function() {
+const SESSION_NOT_OPEN = 'Feedback Session Not Open';
 
-    var textFields = $('div[id^="responsetext-"]');
+$(document).ready(() => {
+    const textFields = $('div[id^="responsetext-"]');
 
     if (typeof richTextEditorBuilder !== 'undefined') {
-        $.each(textFields, function(i, textField) {
-            var id = $(textField).attr('id');
-            var idSuffix = id.match(/^responsetext(.*)$/)[1];
+        $.each(textFields, (i, textField) => {
+            const id = $(textField).attr('id');
+            const idSuffix = id.match(/^responsetext(.*)$/)[1];
 
             /* eslint-disable camelcase */ // The property names are determined by external library (tinymce)
-            richTextEditorBuilder.initEditor('#' + id, {
+            richTextEditorBuilder.initEditor(`#${id}`, {
                 inline: true,
-                fixed_toolbar_container: '#rich-text-toolbar-response-text-container' + idSuffix,
-                setup: function(ed) {
-                    ed.on('keyup', function() {
+                fixed_toolbar_container: `#rich-text-toolbar-response-text-container${idSuffix}`,
+                setup(ed) {
+                    ed.on('keyup', function () {
                         updateTextQuestionWordsCount(id, $(textField).data('lengthTextId'), $(this).data('recommendedText'));
                     });
-                    ed.on('keydown', function() {
+                    ed.on('keydown', function () {
                         updateTextQuestionWordsCount(id, $(textField).data('lengthTextId'), $(this).data('recommendedText'));
                     });
-                    ed.on('init', function() {
+                    ed.on('init', function () {
                         updateTextQuestionWordsCount(id, $(textField).data('lengthTextId'), $(this).data('recommendedText'));
                     });
-                    ed.on('change', function() {
+                    ed.on('change', function () {
                         updateTextQuestionWordsCount(id, $(textField).data('lengthTextId'), $(this).data('recommendedText'));
                     });
-                }
+                },
             });
             /* eslint-enable camelcase */
         });
     }
 
-    $('form[name="form_submit_response"]').submit(function() {
+    $('form[name="form_submit_response"]').submit((e) => {
         formatRubricQuestions();
 
-        var validationStatus = true;
-
-        validationStatus &= validateConstSumQuestions();
-        validationStatus &= validateRankQuestions();
-        validationStatus &= validateAllAnswersHaveRecipient();
+        const validationChecks = [validateConstSumQuestions, validateRankQuestions, validateAllAnswersHaveRecipient];
+        const validationStatus = validationChecks.reduce((acc, f) => acc && f(), true);
```

Comment:
Could simply be `validateConstSumQuestions() && validateRankQuestions() && validateAllAnswersHaveRecipient();`.